packages/documentation/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:lts
+
+# The dockerfile folder has to match the documentation folder name as
+# Docusaurus uses the top folder (documentation) to find the docs foler
+WORKDIR /documentation
+
```

Comment:
Might have to export your `node_modules` bin to container's $PATH. see the client container's Dockerfile. I think it's needed to get the `docusaurus` i'm not too sure. try it with a clean. if this works then it's fine.